charts/pxc-db/README.md

```diff
@@ -101,20 +106,24 @@ The chart can be customized using the following configurable parameters:
-| `haproxy.replicasServiceEnabled`       | Allow disabling k8s service for haproxy-replicas                             | `true` |
 | `haproxy.imagePullSecrets`             | HAProxy Container pull secret                                                | `[]`                                      |
 | `haproxy.configuration`             | User defined HAProxy options according to HAProxy configuration file syntax       | ``     |
-| `haproxy.annotations`             | HAProxy Pod user-defined annotations                                         | `{}` |
 | `haproxy.priorityClassName`       | HAProxy Pod priority Class defined by user                                   |  |
 | `haproxy.runtimeClassName`        | Name of the Kubernetes Runtime Class for HAProxy Pods                        |  |
 | `haproxy.externalTrafficPolicy`   | Desire service to route external traffic for HAProxy to node-local or cluster-wide endpoints  |  |
-| `haproxy.replicasExternalTrafficPolicy` | Desire service to route external traffic for HAProxy replicas to node-local or cluster-wide endpoints  |  |
 | `haproxy.loadBalancerSourceRanges` | Limit which client IP's can access the Network Load Balancer                | `[]` |
+| `haproxy.loadBalancerIP`          | The static IP-address for the load balancer                                  | `` |
 | `haproxy.serviceType`             | Specify what kind of Service you want for HAProxy                            | `ClusterIP` |
+| `haproxy.replicasServiceEnabled`       | Allow disabling k8s service for haproxy-replicas                        | `true` |
 | `haproxy.replicasServiceType`     | Specify what kind of Service you want for HAProxy Replicas                   | `ClusterIP` |
+| `haproxy.replicasExternalTrafficPolicy` | Desire service to route external traffic for HAProxy replicas to node-local or cluster-wide endpoints  |  |
+| `haproxy.replicasServiceAnnotations` | The Kubernetes annotations metadata for the haproxy-replicas Service      | {} |
+| `haproxy.replicasServiceLabels`   | The Kubernetes labels for the haproxy-replicas Service                       | {} |
 | `haproxy.serviceAnnotations`      | Specify service annotations                                                  | `{}` |
+| `haproxy.serviceLabels`           | Specify service labels                                                       | `{}` |
+| `haproxy.annotations`             | HAProxy Pod user-defined annotations                                         | `{}` |
 | `haproxy.labels`                  | HAProxy Pod user-defined labels                                              | `{}` |
 | `haproxy.schedulerName`           | The Kubernetes Scheduler                                                     |      |
 | `haproxy.readinessDelaySec`       | HAProxy Pod delay for readiness probe in seconds                             | `15` |
```

Comment:
replicasLoadBalancerSourceRanges:
#      - 10.0.0.0/8
#    replicasLoadBalancerIP: 127.0.0.1
#    replicasExternalTrafficPolicy: Cluster
Could you please check part about haproxy replicas?